StPicoDstMaker/StPicoDstMaker.cxx

```diff
@@ -77,7 +77,7 @@ StPicoDstMaker::StPicoDstMaker(char const* name) : StMaker(name),
   createArrays();
```

Comment:
Nice! Could you this also fix
https://github.com/rnc-lbl/star-picoDst/blob/66311d02e5ae6cd151794d90511919333a823ca9/StPicoDstMaker/StPicoDstMaker.cxx#L228-L243
https://github.com/rnc-lbl/star-picoDst/blob/66311d02e5ae6cd151794d90511919333a823ca9/StPicoDstMaker/StPicoDstMaker.cxx#L278-L289